RecoMTD/TransientTrackingRecHit/python/MTDTransientTrackingRecHitBuilder_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+MTDTransientTrackingRecHitBuilderESProducer = cms.ESProducer("MTDTransientTrackingRecHitBuilderESProducer",
```

Comment:
please match the file name before ```_cfi``` with the name of the instance defined inside it.
Better yet, define a fillDescriptions